csv_exercise40.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+import csv
+
+imported_data = [fields for fields in csv.reader(open("data_exercise40.txt", newline =''))]
```

Comment:
Consider extracting `"data_exercise40.txt"` as a constant that you pass in to this.